Source/Core/Core/HW/DSPHLE/UCodes/AX.cpp

```diff
@@ -2,14 +2,19 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
-#include "Core/HW/DSPHLE/UCodes/AX.h"
+#include <memory>
+
 #include "Common/ChunkFile.h"
 #include "Common/CommonFuncs.h"
+#include "Common/CommonPaths.h"
 #include "Common/CommonTypes.h"
+#include "Common/File.h"
 #include "Common/FileUtil.h"
 #include "Common/Logging/Log.h"
 #include "Common/MathUtil.h"
+#include "Core/DSP/DSPCore.h"
```

Comment:
This looks like a DSP LLE file, but I'm not sure. Is it fine to include even though this is DSP HLE? I'm using it to get named constants for DSP ROM sizes.